src/components/settings/settings.tsx

```diff
@@ -65,12 +69,149 @@ class Settings extends React.Component<Props, State> {
             </Box>
           </Toolbar>
         </AppBar>
-        <Box p={ 3 }>
-          {/* Settings content here */}
-        </Box>
+        { this.accordionMenu() }
```

Comment:
```suggestion
        { this.renderAccordionMenu() }
```

When methods render something like here - it is good to include "render" in their name